base_layer/core/Cargo.toml

```diff
@@ -30,7 +29,7 @@ tari_comms_dht = { version = "^0.2", path = "../../comms/dht"}
 tari_broadcast_channel = "^0.2"
 tari_shutdown = { version = "^0.0", path = "../../infrastructure/shutdown" }
 tari_mmr = { version = "^0.2", path = "../../base_layer/mmr", optional = true }
-randomx-rs = { version = "0.4.0", optional = true }
+randomx-rs = { version = "0.4.0", optional = true, path = "../../../randomx-rs" }
```

Comment:
```
randomx-rs = { version = "0.4.0", optional = true }
```

Reply:
good catch, forgot to remove this after updating a derive flag.